modular/gater/helper_test.go

```diff
@@ -8,12 +8,13 @@ import (
 )
 
 const (
-	testDomain        = "www.route-test.com"
-	scheme            = "https://"
-	mockBucketName    = "mock-bucket-name"
-	mockObjectName    = "mock-object-name"
-	invalidBucketName = "1"
-	invalidObjectName = ".."
+	testDomain             = "www.route-test.com"
+	scheme                 = "https://"
+	mockBucketName         = "mock-bucket-name"
+	mockObjectName         = "mock-object-name"
+	mockSpecialObjecttName = "?limit=1"
```

Comment:
can we include more special chars in this case?

Reply:
fixed